src/vmm/src/vstate/vcpu/x86_64.rs

```diff
@@ -250,6 +266,8 @@ impl KvmVcpu {
          * meaningful. For SET_MSRS it will then contain good data.
          */
 
+        let tsc_khz = self.get_tsc_khz()?;
```

Comment:
Do we still want to allow snapshots to be created if this fails ?

Reply:
Yes, as we discussed earlier. Fixed it.